Dockerfile

```diff
@@ -8,43 +8,49 @@ RUN apt-get update && apt-get install -y \
 WORKDIR /tmp
 
 # https://github.com/protocolbuffers/protobuf
-ARG PROTOBUF_VERSION
+# renovate: datasource=github-releases depName=protoc packageName=protocolbuffers/protobuf
+ARG PROTOBUF_VERSION=29.3
 RUN curl -sSL "https://github.com/protocolbuffers/protobuf/releases/download/v${PROTOBUF_VERSION}/protoc-${PROTOBUF_VERSION}-linux-x86_64.zip" -o protoc.zip && \
     unzip protoc.zip -d protoc/ && \
     chmod +x ./protoc/bin/protoc
 
 # https://github.com/protocolbuffers/protobuf-javascript
-ARG PROTOBUF_JAVASCRIPT_VERSION
+# renovate: datasource=github-releases depName=protobuf-javascript packageName=protocolbuffers/protobuf-javascript
+ARG PROTOBUF_JAVASCRIPT_VERSION=3.21.4
 RUN curl -sSL "https://github.com/protocolbuffers/protobuf-javascript/releases/download/v${PROTOBUF_JAVASCRIPT_VERSION}/protobuf-javascript-${PROTOBUF_JAVASCRIPT_VERSION}-linux-x86_64.zip" \
     -o protoc-gen-js.zip && \
     unzip protoc-gen-js.zip -d protoc-gen-js/ && \
     chmod +x protoc-gen-js/bin/protoc-gen-js
 
 # https://github.com/grpc/grpc-web
 WORKDIR /tmp
-ARG GRPC_WEB_VERSION
+# renovate: datasource=github-releases depName=grpc-web packageName=grpc/grpc-web
+ARG GRPC_WEB_VERSION=1.5.0
 RUN curl -sSL "https://github.com/grpc/grpc-web/releases/download/${GRPC_WEB_VERSION}/protoc-gen-grpc-web-${GRPC_WEB_VERSION}-linux-x86_64" -o /usr/local/bin/protoc-gen-web-grpc && \
     chmod +x /usr/local/bin/protoc-gen-web-grpc
 
 # https://pkg.go.dev/google.golang.org/protobuf/cmd/protoc-gen-go
-ARG PROTOBUF_GO_VERSION
+# renovate: datasource=go depName=protoc-gen-go packageName=google.golang.org/protobuf/cmd/protoc-gen-go
+ARG PROTOBUF_GO_VERSION=1.36.3
 RUN GOBIN=/usr/local/bin go install google.golang.org/protobuf/cmd/protoc-gen-go@v${PROTOBUF_GO_VERSION}
 
 # https://pkg.go.dev/google.golang.org/grpc/cmd/protoc-gen-go-grpc
-ARG GRPC_GO_VERSION
+# renovate: datasource=go depName=protoc-gen-go packageName=google.golang.org/grpc/cmd/protoc-gen-go-grpc
```

Comment:
Should this one be `depName=protoc-gen-go-grpc` because the one above it is `depName=protoc-gen-go`?

Reply:
Oops! Yes! Thanks for reviewing and catching that.